test/acceptance/display_matching_invocations_alongside_expectations_test.rb

```diff
@@ -15,88 +15,45 @@ def teardown
     teardown_acceptance_test
   end
 
-  def test_should_display_return_values_of_matching_invocations
+  def test_should_display_results_of_matching_invocations_alongside_expectations_in_verbose_mode
     test_result = run_as_test do
       foo = mock('foo')
       foo.expects(:bar).with(1).returns('a')
-      foo.stubs(:bar).with(2).returns('b').then.returns('c')
+      foo.stubs(:bar).with(any_parameters).returns('f').raises(StandardError)
 
-      2.times { foo.bar(2) }
-      foo.bar(3)
+      foo.bar(1, 2) { |_ignored| }
```

Comment:
Minor: I'm guessing the underscore prefix was added here (and in other lines in this test) to fix a Rubocop violation. It would be nice if this was done in the first commit where it was needed.

Reply:
OK. I know it's a bit pedantic, but ideally I'd like Rubocop and all the tests to pass for each commit. So as long as what you come up with achieves that, I'll be happy!

Reply:
> I respect pedantics. I'm usually a pedant myself.

😄 

> No failing commits is what I'm striving for. Sometimes it's a bit hard to tell, because although I make sure I commit only when all checks have passed locally, I don't push every commit. So, I may not realize if some intermediate commits would fail on travis for some particular setup. In most cases, I think it should be fine because when a travis build fails, I fix it using a git fixup.
> 
> Do you have a better way of ensuring this?

That's all good to know. I usually run `rake test` *locally* (which includes Rubocop checks) after every commit typically using the latest Ruby version. This runs very fast and so doesn't hinder me as much as waiting for a Travis CI build, but still gives me a lot of confidence I haven't broken anything.

Reply:
I suppose I could've been clearer. I don't commit unless rake test with my version of ruby has succeeded either. So, we have have the same process there. It's more the case of travis builds failing with other ruby versions that I wouldn't know about until I push, which I usually do only with a bunch of commits, which means that some intermediate commits could be failing for some (other) ruby version.

Reply:
Cool 😎

Reply:
Resolved in https://github.com/freerange/mocha/pull/394#issuecomment-551927383.